docs/UserGuide.md

```diff
@@ -280,12 +303,11 @@ Examples:
 
 Possible Usage:
 
-* You can use `edit` to make any changes to a student's relevant information after you has mistyped any.
+* You can use `edit` to make changes to any typo in a student's information.
```

Comment:
Would  'make changes to any mistake made in a student's information', sound better? Typo feels a bit informal.

Reply:
good point